src/promptflow/promptflow/_sdk/entities/_connection.py

```diff
@@ -618,6 +625,113 @@ def _get_schema_cls(cls):
         return FormRecognizerConnectionSchema
 
 
+class CustomStrongTypeConnection(_Connection):
+    """Custom strong type connection.
+
+    :param configs: The configs kv pairs.
+    :type configs: Dict[str, str]
+    :param secrets: The secrets kv pairs.
+    :type secrets: Dict[str, str]
+    :param name: Connection name
+    :type name: str
+
+    This connection type should not be used directly. Below is an example of how to use CustomStrongTypeConnection:
```

Comment:
This description should happen before the param lists. 

Please follow this guide: 
https://github.com/microsoft/promptflow/blob/main/docs/dev/documentation_guidelines.md